### DISCUSSION TITLE
Win11 DPI缩放值变化了，怎么才能调整当前窗口中的界面？

### DISCUSSION THREAD

**Discussion by happyscene** [netease-im/NIM_Duilib_Framework#495] [Q&A]
两个屏幕，屏幕1的DPI缩放值 200%，屏幕2的DPI缩放值 150%。把窗口从屏幕1拖拽到屏幕2，怎么调整窗口的界面呢？
能否根据当前的DPI值把界面组件的尺寸调整一遍？

**Comment by nmgwddj** [comment]
能力上并没有实现动态 DPI change 事件，这个是已知的不足。您可以考虑设置应用的 DPI 变更由系统决定而不是应用决定，这样可以临时满足需求，但可能会出现一些界面缩放、拉伸模糊的问题。

**Reply by happyscene** [reply]
谢谢回复。如果有这个功能就完美了，不知道有做这个功能的计划吗？

**Comment by nmgwddj** [comment]
> 谢谢回复。如果有这个功能就完美了，不知道有做这个功能的计划吗？

目前暂无更新 duilib 的计划，包括新需求和一些低优先级的缺陷。
